authenticatorbridge/src/test/java/com/bitwarden/authenticatorbridge/manager/AuthenticatorBridgeManagerTest.kt

```diff
@@ -87,6 +93,26 @@ class AuthenticatorBridgeManagerTest {
         assertEquals(AccountSyncState.AppNotInstalled, manager.accountSyncStateFlow.value)
     }
 
+    @Test
+    fun `initial AccountSyncState should be OsVersionNotSupported when OS level is below S`() {
+        every { isBuildVersionBelow(Build.VERSION_CODES.S) } returns true
+        val manager = AuthenticatorBridgeManagerImpl(
+            context = context,
+            connectionType = AuthenticatorBridgeConnectionType.DEV,
+            symmetricKeyStorageProvider = fakeSymmetricKeyStorageProvider,
+            callbackProvider = testAuthenticatorBridgeCallbackProvider,
+            processLifecycleOwner = fakeLifecycleOwner,
+        )
+        assertEquals(AccountSyncState.OsVersionNotSupported, manager.accountSyncStateFlow.value)
+    }
+
+    @Test
+    fun `onStart when OS level is below S should set state to OsNotSupported`() {
```

Comment:
```suggestion
    fun `onStart when OS level is below S should set state to OsVersionNotSupported`() {
```

Reply:
Thank you. 

596b616e4af3a52754358d3262b4544e37fbd023